site/config.yaml

```diff
@@ -37,6 +37,7 @@ params:
   docs_versioning: true
   docs_versions:
     - main
+    - v1.18.3
```

Comment:
Looks like the version drop-down renders things per the order in this list, so let's move this down to below v1.19.0